.github/workflows/validate-examples.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate AsyncAPI Examples
+
+on:
+  push:
+    branches:
+      - script-ci-spec-validation # added temporarily for testing purpose
+      # - master
+
+  pull_request_target:
+    types: [opened, reopened, synchronize, edited, ready_for_review]
+
+jobs:
+  validate-examples:
+    name: Validate Spec Examples
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+      - name: Install AsyncAPI CLI
+        run: npm install -g @asyncapi/cli
```

Comment:
bo it uses latest CLI - advantage is that all latest features will be used but obvious disadvantage is randomly failing CI cause of some breaking change in validation, or some new validation enabled.

yeah, not so obvious in this case 😕

Reply:
so should be use a specific version of the CLI? :eyes:

Reply:
If I can give my opinion here, I think using latest is a good option. We are always testing on the last version, supporting last spec schemas, etc + if something is wrong with the CLI, we are gonna be one of the first who notice.

Reply:
ping @derberg  @AnimeshKumar923

Reply:
Ping @derberg 👋

Reply:
no strong opinion really, there are pros and cons for using latest and not using latest. We just need to be aware that if we decide to use latest - then on PRs we can have random fails from time to time and we need to support contributors

Reply:
Agree. I suggest then we can try with this but opened to change if needed at any time.